### PR TITLE
[SF7] Add Symfony 7 support for MoneyBundle

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
|-----------------|-----
| Branch?         | symfony-7
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | replaces #16387
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/MoneyBundle/Twig/ConvertMoneyExtension.php [hunk 1]
```diff
@@ -13,20 +13,20 @@
...
 
-use Sylius\Bundle\MoneyBundle\Templating\Helper\ConvertMoneyHelperInterface;
+use Sylius\Component\Currency\Converter\CurrencyConverterInterface;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;
 
 final class ConvertMoneyExtension extends AbstractExtension
 {
-    public function __construct(private ConvertMoneyHelperInterface $helper)
+    public function __construct(private CurrencyConverterInterface $currencyConverter)
```
Comment (loic425) [inline src/Sylius/Bundle/MoneyBundle/Twig/ConvertMoneyExtension.php:22]: I think you shoud add a note on the UPGRADE guide for that too.

Comment (diimpp) [reply src/Sylius/Bundle/MoneyBundle/Twig/ConvertMoneyExtension.php:22]: And for `src/Sylius/Bundle/MoneyBundle/Twig/FormatMoneyExtension.php` as well

File: UPGRADE-2.0.md [hunk 2]
```diff
@@ -180,6 +180,15 @@
       `Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository` to
       `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAssociationRepository`
 
+* The following helper classes and interfaces have been removed:
```
Comment (diimpp) [inline UPGRADE-2.0.md]: If this list point is general purpose removal mention, then it should be like so,
```suggestion
* The following classes and interfaces have been removed:
```

Comment (GSadee) [reply UPGRADE-2.0.md:183]: For now it is only for helpers so I leave it as it is, in the nearest future we will definitely go through all the changes in 2.0 and update this file 🖖🏻

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @GSadee!

**Pull request merged**